packages/step/src/StepItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+
+const StepItem = ({ children, className, active, ...props }) => {
+  const classNames = classnames('step-item', className, {
```

Comment:
I prefer this component to be named as `Item` instead. And inside `Step` component add an alias to `Step.Item`. What do you think?

Reply:
Sounds good!